libs/driver-adapters/executor/src/qc-test-worker/worker.ts

```diff
@@ -98,25 +98,46 @@ if (!parentPort) {
 
 setupDefaultPanicHandler()
 
+type RpcResponse =
+  | null
```

Comment:
`null` is used for no return value, otherwise it conflicts with the decoding of `RpcError`

Reply:
Shouldn't we instead ensure we always return a proper `Result` or `Error`?